README.md

```diff
@@ -41,6 +41,7 @@ export MEMBER_PRIV_KEY=$ORACLE_PRIVATE_KEY_0X_PREFIXED
 export POOL_CONTRACT=0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84
 export STETH_PRICE_ORACLE_CONTRACT=0x3a6bd15abf19581e411621d669b6a2bbe741ffd6
 export STETH_CURVE_POOL_CONTRACT=0xDC24316b9AE028F1497c275EB9192a3Ea0f67022
+export WITHDRAWAL_VAULT_CONTRACT=0xB9D7934878B5FB9610B3fE8A5e441e8fad7E293f
```

Comment:
I would suggest getting the address from Lido contract -> withdrawal credentials

Reply:
fixed